packages/sanity/src/structure/panes/document/useDocumentTitle.ts

```diff
@@ -50,6 +51,15 @@ export function useDocumentTitle(): UseDocumentTitle {
     }
   }
 
+  // in cases where the displayed document has been slated to be unpublished
+  // then the displayed title should be the name of the document
+  if ('_system' in displayed && (displayed as Partial<SanityDocument>)._system?.delete) {
```

Comment:
I considered using the condition `isGoingToUnpublish(displayed as SanityDocumentLike)` (see [here](https://github.com/sanity-io/sanity/blob/sapp-2635-follow-up/packages/sanity/src/core/releases/util/isGoingToUnpublish.ts#L11)) but this felt safer to do it this way since the implementation woudl have had to be with a few more castings that I don't quite undestand _why_ it's needed except the code editor screams otherwise

```suggestion
  if (isGoingToUnpublish(displayed as SanityDocument)) {
    return {
      error: undefined,
      title: (displayed as SanityDocument).name as string,
    }
  }
```